examples/decoding/plot_receptive_field.py

```diff
@@ -160,23 +160,26 @@
 
 
 ###############################################################################
-# Create and fit a stimulus reconstruction model (a.k.a. backward model)
-# ----------------------------------------------------------------------
+# Create and fit a stimulus reconstruction model
+# ----------------------------------------------
 #
-# We will now construct a model to find the linear relationship between the
-# speech signal and a time-delayed version of the EEG. This can be advantageous
-# as we exploit all of the available neural data in a multivariate context,
-# compared to the forward case which treats each M/EEG channel as an
-# independent feature. Therefore, low SNR stimuli such as speech will likely
-# have better backward models.
-
-# We use the same lags as in [1]_
+# We will now demonstrate another use case for the ReceptiveField module as we
+# try to predict the stimulus activity from the EEG data. This is known in the
+# literature as a decoding, or stimulus reconstruction model [1]_. A decoding
+# model aims to find the linear relationship between the speech signal and a
```

Comment:
strictly speaking a decoding model doesn't _have_ to find a linear relationship...maybe just call it a "linear decoding model"? (though I agree with you that 95% of the time this is what people do in stimulus reconstruction, at least for now)